packages/calcite-components/src/components/card/card.scss

```diff
@@ -7,6 +7,20 @@
  * @prop --calcite-card-border-color: Specifies the border color of the component.
  * @prop --calcite-card-box-shadow: Specifies the box shadow of the component.
  * @prop --calcite-card-corner-radius: Specifies the corner radius of the component.
+ * @prop --calcite-card-label-text-color: defines the label text colorin the component.
+ * @prop --calcite-card-checkbox-background-color-checked: defines the checkbox background color when checked in the component.
+ * @prop --calcite-card-checkbox-background-color: defines the checkbox background color in the component.
+ * @prop --calcite-card-checkbox-icon-color: defines the checkbox icon color in the component.
+ * @prop --calcite-card-checkbox-shadow-checked: defines the checkbox shadow when checked in the component.
+ * @prop --calcite-card-checkbox-shadow-hover: defines the checkbox shadow when hovered in the component.
+ * @prop --calcite-card-checkbox-shadow-invalid: defines the checkbox shadow when invalid in the component.
```

Comment:
These checkbox ones should be introduced as deprecated as possible as they are no longer used once Card Group lands. That PR has tokens for the replacement. 

The checkbox also cannot be set to invalid or have a label currently as it is rendered inside the component and a user cannot set invalid on either the checkbox or the card.